Simplenote/NoticeFactory.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+struct NoticeFactory {
+    static let linkCopied = Notice(message: NoticeFactory.copyLinkMessage, action: nil)
+    static let publishing = Notice(message: NSLocalizedString("Publishing...", comment: "Notice of publishing action"), action: nil)
+    static let unpublishing = Notice(message: NSLocalizedString("Unpublishing...", comment: "Notice of unpublishing action"), action: nil)
```

Comment:
Nipticky: For consistency's reasons, maybe all of the notices should be `static func` ?